cf_units/tests/test_coding_standards.py

```diff
@@ -109,7 +109,7 @@ def test_license_headers(self):
             )
 
         failed = False
-        for fname, last_change in sorted(last_change_by_fname.items()):
+        for fname, _last_change in sorted(last_change_by_fname.items()):
```

Comment:
Should maybe just be

```suggestion
        for fname in sorted(last_change_by_fname):
```
?